src/config.rs

```diff
@@ -1,6 +1,11 @@
+#![allow(unused_must_use)]
```

Comment:
I don't think this is warranted.
Sorry I hadn't noticed this earlier. Everything else is good!